cfgmgr/teammgr.cpp

```diff
@@ -163,6 +163,7 @@ void TeamMgr::cleanTeamProcesses(int signo)
 {
     pid_t pid = 0;
 
+    SWSS_LOG_NOTICE("Cleaning up LAGs during shutdown...");
```

Comment:
I don't think this method has `SWSS_LOG_ENTER();` either.

Reply:
Added. Thanks!